src/main/java/io/reactivex/CompletableEmitter.java

```diff
@@ -17,7 +17,7 @@
 import io.reactivex.functions.Cancellable;
 
 /**
- * Abstraction over a RxJava CompletableObserver that allows associating
+ * Abstraction over an RxJava {@link CompletableObserver} that allows associating
```

Comment:
Yes because it's pronounced differently you can say `an` and it also does sound better. Jake taught me that one too